src/main/kotlin/lotto/LottoGame.kt

```diff
@@ -0,0 +1,17 @@
+package lotto
+
+class LottoGame(val playersTickets: List<Lotto>, val winningNumbers: Lotto, val bonusNumber: Int) {
+    val results: MutableMap<Rank, Int> = mutableMapOf()
+    var winningAmount = 0
+
+    fun play() {
+        winningAmount == 0
```

Comment:
maybe... `winningAmount = 0` ?

Reply:
It looks like a new instance will be allocated every time a new game starts.
Since `winningAmount` and `results` will also be freshly initialized each time, there doesn't seem to be a need to initialize them in every `play()` call!